tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -14,6 +14,7 @@ use strict;
 use warnings;
 use File::Basename;
 use testapi;
+use virt_utils "clean_up_red_disks";
```

Comment:
use single quote `'clean_up_red_disks'`

Reply:
Using single quote does not match the already existing coding style use base "reboot_and_wait_up";

Reply:
the coding style is single to quote string without variables

Reply:
Yes, I think it is not written down anywhere but IMHO so far the best practice was to use single quotes by default unless you want to evaluate variables within strings